### ISSUE TITLE
Apply filter templates to users

### ISSUE THREAD

**Issue by xrmprofessional** [MscrmTools/MscrmTools.SyncFilterManager#4]
When I try to apply Outlook template to one ore more users I get this error message:

"Error while applying filters to users. Expected non-empty Guid."

I hadn't this issue with the previous version of this plugin. Can I download somewhere the old version until this is fixed?

Thanks in advance!

**Comment by MscrmTools** [comment]
Yes, you can get it there : https://www.nuget.org/packages/MscrmTools.SyncFilterManager/1.2017.3.1

rename the .nupkg to .zip, and find the file from the Plugins folder. paste it to your XrmToolBox plugins folder
